tests/e2e/config/global-teardown.js

```diff
@@ -38,6 +41,7 @@ module.exports = async ( config ) => {
 			consumerTokenCleared = true;
 			break;
 		} catch ( e ) {
+			console.log( e );
```

Comment:
I agree with logging the error, but maybe we should be either using `console.error` or annotating the log somehow? e.g. `console.error( 'Failed to clear consumer token because:', e );`